ThirdPartyModules/Listrak/Remarketing.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2017-2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\ThirdPartyModules\Listrak;
+
+use Bolt\Boltpay\Helper\Bugsnag;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+class Remarketing
+{
+    const BOLT_LISTRAK_FILED_ID = 'bolt_listrak';
+    const XML_PATH_LISTRAK_REMARKETING_TRACK_WITH_LISTRAK = 'remarketing/onescript/enabled';
+    const XML_PATH_LISTRAK_REMARKETING_MERCHANT_ID = 'remarketing/onescript/merchant_id';
+
+    /**
+     * @var Bugsnag
+     */
+    private $bugsnagHelper;
+
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $scopeConfigInterface;
+
+    /**
+     * Remarketing constructor.
+     * @param Bugsnag $bugsnagHelper
+     * @param ScopeConfigInterface $scopeConfigInterface
+     */
+    public function __construct(
+        Bugsnag $bugsnagHelper,
+        ScopeConfigInterface $scopeConfigInterface
+    )
+    {
+        $this->scopeConfigInterface = $scopeConfigInterface;
+        $this->bugsnagHelper = $bugsnagHelper;
+    }
+
+    /**
+     * Add js to capture email to Listrak whenever customer enter email in the Botl modal
+     *
+     * @param string $result
+     * @return string
+     */
+    public function getOnEmailEnter($result)
```

Comment:
Could we, just in case, add this integration with a feature flag that is enabled by default?

Reply:
I've updated the code. Can you re-check?